apps/api/validators.py

```diff
@@ -1,12 +1,384 @@
-from django.core.exceptions import ValidationError
+'''Validates if the file given is a csv file'''
 
 import csv
+from re import match
+from django.core.exceptions import ValidationError
+
+
+class ColumnError(Exception):
+    '''Exception raised if the number of columns in the csv file header does
+    not match the rest'''
+    pass
+
+
+class MultipleUtoridOccurences(Exception):
+    '''Exception raised if the same utor id occurs multiple times in a file
+    '''
+    pass
+
+
+class InvalidUtorid(Exception):
+    '''Utor id not valid. It has to be 8 characters long and only contain
+    alphanumberic symbols
+    '''
+    pass
+
+
+class InvalidData(Exception):
+    '''If the data at a given column does not have the proper structure or
+    seem valid
+    '''
+    pass
+
+
+def _is_lowercase_alphabet(value):
```

Comment:
match() returns None is false and cause function to return AttributeError: ‘NoneType’ object has no attribute ‘group’

Possible solution:

```
	_is_lowercase_alphabet(value):
		return (value.isalpha() and value.islower())
```
Ex. _is_lowercase_alphabet(“”)
False